logcat/src/commonMain/kotlin/logcat/PrintLogger.kt

```diff
@@ -3,11 +3,15 @@ package logcat
 /**
  * A [LogcatLogger] that always logs and delegates to [println] concatenating
  * the tag and message, separated by a space. Alternative to [AndroidLogcatLogger]
- * when running on a JVM.
+ * when running on a JVM/JS/Native.
  */
 object PrintLogger : LogcatLogger {
```

Comment:
It looks like the "other platforms" only get println. Consider iOS and JS specific loggers (https://github.com/touchlab/Kermit/issues/183, https://github.com/touchlab/Kermit/blob/main/kermit/src/jsMain/kotlin/co/touchlab/kermit/ConsoleWriter.kt).

Reply:
Lots of good KMP stuff to learn from Kermit, for sure. Unfortunately a few things I wish were different (lack of auto tag and requiring devs to make a decision about severity instead of making that optional. See here https://github.com/square/logcat#motivations)

Reply:
Well, I mean, I'm not saying "use Kermit", I'm just saying if it's multiplatform, I'd add some platform-specific loggers. The first link is to a thread where we took inspiration from Napier, which is another KMP logger. I'll definitely take a look at the motivations, as we're sort of redesigning Kermit right now. Tagging is tricky, for sure.

Reply:
Certainly seems like we could make a JS specific logger that uses the console API.  I'll give it a try.